examples/src/main/java/com/flipkart/gjex/examples/helloworld/filter/LoggingGrpcFilter.java

```diff
@@ -15,31 +15,33 @@
  */
 package com.flipkart.gjex.examples.helloworld.filter;
 
-import com.flipkart.gjex.core.filter.Filter;
-import com.flipkart.gjex.core.logging.Logging;
+import com.flipkart.gjex.core.filter.ResponseParams;
+import com.flipkart.gjex.core.filter.grpc.GrpcFilter;
+import com.flipkart.gjex.core.filter.RequestParams;
 import com.google.protobuf.GeneratedMessageV3;
 import io.grpc.Metadata;
 
 import javax.inject.Named;
 
 /**
- * An implementation of the {@link Filter} interface as example that simply logs Request information
+ * An implementation of the {@link GrpcFilter} interface as example that simply logs Request information
  * @author regu.b
  */
-@Named("LoggingFilter")
-public class LoggingFilter<Req extends GeneratedMessageV3, Res extends GeneratedMessageV3> implements Filter<Req, Res>, Logging {
+@Named("LoggingGrpcFilter")
+public class LoggingGrpcFilter<Req extends GeneratedMessageV3,
```

Comment:
This can remain `LoggingFilter`

Lets not change the grpc filter names. Gjex is major for grpc and http is provided for migration period. In long term i don't see the need for having this explicit name in filter.s

Reply:
done